cmd/rapg/main.go

```
@@ -268,6 +329,109 @@ func unlockVault() {
+	}
+
+	out, n := redact.Redact(string(input), secrets)
+	fmt.Print(out)
+	fmt.Fprintf(os.Stderr, "[rapg] redacted %d distinct value(s) from %d candidate(s)\n", n, len(secrets))
+}
+
+// printSessions writes a tab-aligned, human-readable rendering of the
+// session log to stdout. One line per session, ordered oldest to newest.
+func printSessions(sessions []audit.Session) {
+	tw := tabwriter.NewWriter(os.Stdout, 0, 4, 2, ' ', 0)
+	defer tw.Flush()
```

golangci-lint (2.12.1)

[error] Error return value of `tw.Flush` is not checked

(errcheck)